src/python/doc/ls_simplex_tree_tflow_itf_ref.rst

```diff
@@ -0,0 +1,64 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+TensorFlow layer for lower-star persistence on simplex trees
+############################################################
+
+.. include:: differentiation_sum.inc
+
+Example of gradient computed from lower-star filtration of a simplex tree
+-------------------------------------------------------------------------
+
+.. testcode::
+
+    from gudhi.tensorflow import *
```

Comment:
Maybe to be reworked according to [this discussion](https://github.com/GUDHI/gudhi-devel/pull/411#discussion_r742337781)
```suggestion
    from gudhi.tensorflow import LowerStarSimplexTreeLayer
```

Reply:
Thanks! Done.